### PR TITLE
Add the ability to commit and tag after bump

### PR DESCRIPTION
**Pull request by princemaple**
:koala: 

It delegates to shell command execution. The reasons I think it's OK to do so:

1. this is opt in
2. this is not on server side

cc @zendesk/quokka

### CODE REVIEW

File: lib/zendesk_apps_tools/bump.rb [hunk 1]
```diff
@@ -16,16 +23,38 @@ def major
...
+
+    private
+
+    def post_actions
+      return tag if options[:tag]
+      commit if options[:commit]
+    end
+
+    def commit
+      `git commit -am #{commit_message}`
```
Comment (itirkarp) [inline lib/zendesk_apps_tools/bump.rb]: This will not add untracked / new files. Will that be a problem?

Comment (princemaple) [inline lib/zendesk_apps_tools/bump.rb]: Nope, it's meant to commit the change on manifest.

**Comment by itirkarp** [comment]
Looks ok. Hope it works :+1:

**Comment by princemaple** [comment]
It does, I have been using it already.

**Comment by tobico** [comment]
:+1:

**Pull request merged**